wordpress-mobile/WordPress-Android
# Issue/update uihelpers

Moves `setTextOrHide` and `setImageOrHide` to `UiHelpers`.

Note: I haven't refactored `BlockListItemViewHolder` since they use two variables (resId: Int?, text: String?) instead of UiString. We might want to consider refactoring that as well in the future. Wdyt?

To test:
- make sure the app is buildable
- open activity log
- open blog posts
- walk through site creation flow

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
